middlewares/replace_path_regex_test.go

```diff
@@ -0,0 +1,71 @@
+package middlewares
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/containous/traefik/testhelpers"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestReplacePathRegex(t *testing.T) {
+	testCases := []struct {
```

Comment:
Could add a case with invalid regexp?

Reply:
@ldez Done.

Reply:
Ping @ldez 
I added test case with invalid regexp, but the changes wasn't recognized by GitHub.